extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -187,15 +187,15 @@ export class BuildHelper {
 		// since SDK style projects don't to specify the targets path, just where the system dacpacs are
 		if (utils.getAzdataApi()) {
 			if (sqlProjStyle === mssql.ProjectType.SdkStyle) {
-				return ` build ${projectPath} /p:NetCoreBuild=true /p:SystemDacpacsLocation=${buildDirPath}`;
+				return `/p:NetCoreBuild=true /p:SystemDacpacsLocation=${buildDirPath} ${constants.detailedVerbose}`;
```

Comment:
Should the verbosity be something user can control?

Reply:
Yes, we can add the verbosity to the tasks.json, where user can control it, but that won't be adding to the Build and BuildWithCodeAnalysis options as we are preparing the build command within the code.